com.unity.visualeffectgraph/Runtime/Utilities/Playables/VisualEffectActivationDeprecated/VisualEffectActivationBehaviour.cs

```diff
@@ -0,0 +1,44 @@
+#if VFX_HAS_TIMELINE
+using System;
+using UnityEngine;
+using UnityEngine.Timeline;
+using UnityEngine.Playables;
+using UnityEngine.VFX;
+using UnityEngine.VFX.Utility;
+
+[Serializable]
+class VisualEffectActivationBehaviour : PlayableBehaviour
+{
+    [Serializable]
+    public enum AttributeType
+    {
+        //Actually enum values are synchronized with VFXValueType
```

Comment:
Why do we need another enum then? Seems potentially error prone to maintain

Reply:
Mmh it's in a "deprecated" file so probably no relevant anymore

Reply:
I actually moved it in this PR to deprecated, it's needed to be able to deserialize old data before sanitization.

Reply:
By the way, the new implementation doesn't require a serialization of an enum, it uses _SerializableReference_: https://github.com/Unity-Technologies/Graphics/blob/11af1e72761af53f27294b3ee93945156445b105/com.unity.visualeffectgraph/Runtime/Utilities/Playables/VisualEffectControl/VisualEffectControlPlayableBehaviour.cs#L26